nautobot/extras/datasources/git.py

```diff
@@ -554,6 +570,155 @@ def delete_git_config_contexts(repository_record, job_result, preserve=(), prese
                 )
 
 
+#
+# Config Context Schemas
+#
+
+
+def refresh_git_config_context_schemas(repository_record, job_result, delete=False):
+    """Callback function for GitRepository updates - refresh all ConfigContextSchema records managed by this repository."""
+    if "extras.configcontextschema" in repository_record.provided_contents and not delete:
+        update_git_config_context_schemas(repository_record, job_result)
+    else:
+        delete_git_config_context_schemas(repository_record, job_result)
+
+
+def update_git_config_context_schemas(repository_record, job_result):
+    """Refresh any config context schemas provided by this Git repository."""
+    config_context_path = os.path.join(repository_record.filesystem_path, "config_context_schemas")
```

Comment:
Can we change `config_context_path` to either `config_context_schema_path` or simply `schema_path` throughout this function?